### DISCUSSION TITLE
请问下如何支持阻塞同步的文件加载？

### DISCUSSION THREAD

**Discussion by yangshuang-developer** [layabox/LayaAir#1363] [General]
`export function loadRes(url:string){
    return new Promise(resolve=>{
        Laya.loader.load(url, Handler.create(this,()=>{
            resolve();}));
    });
}`
`    async testLoad() {
        await loadRes(getResPath('monkey0.png'));`

这样结果不是同步的，这个有办法吗？

**Comment by xiaoguzhu** [comment]
同步加载在web环境是无法支持的
